pkgs/development/tools/sunxi-tools/default.nix

```diff
@@ -1,25 +1,25 @@
-{ stdenv, fetchFromGitHub, pkgconfig, libusb }:
+{ stdenv, fetchFromGitHub, pkgconfig, libusb, zlib }:
 
 stdenv.mkDerivation {
-  name = "sunxi-tools-1.3";
+  name = "sunxi-tools-20171130";
 
   src = fetchFromGitHub {
     owner = "linux-sunxi";
     repo = "sunxi-tools";
-    rev = "be1b4c7400161b90437432076360c1f99970f54f";
-    sha256 = "02pqaaahra4wbv325264qh5i17mxwicmjx9nm33nw2dpmfmg9xhr";
+    rev = "5c1971040c6c44caefb98e371bfca9e18d511da9";
+    sha256 = "0qzm515i3dfn82a6sf7372080zb02d365z52bh0b1q711r4dvjgp";
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ libusb ];
+  buildInputs = [ libusb zlib ];
 
   buildPhase = ''
-    make all misc
+    make tools misc
   '';
 
   installPhase = ''
     mkdir -p $out/bin
-    cp bin2fex fex2bin phoenix_info sunxi-bootinfo sunxi-fel sunxi-fexc sunxi-nand-part sunxi-pio $out/bin
+    cp bin2fex fex2bin phoenix_info sunxi-bootinfo sunxi-fel sunxi-fexc sunxi-nand-image-builder sunxi-nand-part sunxi-pio $out/bin
```

Comment:
Could you see if the default ```make install``` installPhase works? That usually makes maintanence & updates easier.

Reply:
Yeah it does. 714aff4004cbd8396624f798a485f2d724d1233c